### ISSUE TITLE
Deleted targets

### ISSUE THREAD

**Issue by Lorious** [wowsims/wotlk#3371]
**Describe the bug**

Managed to delete all targets in advanced options in settings section.
I am now unable to add any and just keep getting errors about not having a target when I'm running the sim. Can't use the premade bosses either will not be recognized.

**To Reproduce**

**Expected behavior**

**Sim Links and Screenshots**
If applicable, add links to your sim settings or screenshots to help explain your problem.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Tharre** [comment]
What happens if you use the "New Target" button, do you get any errors?

As a temporary workaround you can restore to defaults which should fix it.

**Comment by Lorious** [comment]
Solved it by going incognito and linking myself a new set.
I wasn't able to use the reset function and refreshing the page didn't work
either, but hey we got up and running again. I'm happy!
If you feel like you want to look into this then just get annoyed by the
fact that you are unable to recreate what my guilde claims he simmed and
spam the cross button in encounter to get rid of the extra bosses you added
in advanced settings just to see if multiple target changed anything, it
didn't.

Med vänlig hälsning
/Jon Enberg
